### PR TITLE
Add CI tests for 4.13 IBMCloud

### PR DESCRIPTION
**Pull request by cappetersson**

### CODE REVIEW

**Comment by cappetersson** [comment]
/pj-rehearse pull-ci-openshift-hypershift-release-4.13-e2e-ibmcloud-iks

File: ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.13.yaml [hunk 1]
```diff
@@ -108,22 +108,20 @@ tests:
   run_if_changed: ^api/
   steps:
     env:
-      BOM_VERSION: 4.10.26_openshift
+      BOM_VERSION: 4.13.0_openshift
       MANAGED_CLUSTER_TYPE: iks
       MANAGEMENT_CLUSTER_KUBECONFIG: /etc/ibmcloud/secrets/iks.kubeconfig
```
Comment (hasueki) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.13.yaml:113]:
```suggestion
      MANAGEMENT_CLUSTER_KUBECONFIG: /etc/ibmcloud/secrets/iks-release.kubeconfig
```

File: ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.13.yaml [hunk 2]
```diff
@@ -108,22 +108,20 @@ tests:
   run_if_changed: ^api/
```
Comment (hasueki) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.13.yaml:108]:
```suggestion
  skip_if_only_changed: (^(docs|examples|enhancements)/)|(^[A-Z]+\.md$)|((^|/)OWNERS$)
```

File: ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.13.yaml [hunk 3]
```diff
@@ -108,22 +108,20 @@ tests:
...
   steps:
     env:
-      BOM_VERSION: 4.10.26_openshift
+      BOM_VERSION: 4.13.0_openshift
       MANAGED_CLUSTER_TYPE: iks
       MANAGEMENT_CLUSTER_KUBECONFIG: /etc/ibmcloud/secrets/iks.kubeconfig
-      TEST_TYPE: main
-      VERIFY_COMPLIANCE_OPERATOR_SCAN: "false"
+      TEST_TYPE: release
     workflow: hypershift-ibmcloud-e2e
```
Comment (hasueki) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.13.yaml:115]:
```suggestion
    workflow: hypershift-ibmcloud-e2e-release
```

File: ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.13.yaml [hunk 4]
```diff
@@ -108,22 +108,20 @@ tests:
...
 - as: e2e-ibmcloud-roks
   optional: true
   run_if_changed: ^api/
   steps:
     env:
-      BOM_VERSION: 4.11.0_openshift
+      BOM_VERSION: 4.13.0_openshift
       MANAGED_CLUSTER_TYPE: openshift
       MANAGEMENT_CLUSTER_KUBECONFIG: /etc/ibmcloud/secrets/roks.kubeconfig
```
Comment (hasueki) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.13.yaml:123]:
```suggestion
      MANAGEMENT_CLUSTER_KUBECONFIG: /etc/ibmcloud/secrets/roks-release.kubeconfig
```

File: ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.13.yaml [hunk 5]
```diff
@@ -108,22 +108,20 @@ tests:
...
+      BOM_VERSION: 4.13.0_openshift
       MANAGED_CLUSTER_TYPE: iks
       MANAGEMENT_CLUSTER_KUBECONFIG: /etc/ibmcloud/secrets/iks.kubeconfig
-      TEST_TYPE: main
-      VERIFY_COMPLIANCE_OPERATOR_SCAN: "false"
+      TEST_TYPE: release
     workflow: hypershift-ibmcloud-e2e
 - as: e2e-ibmcloud-roks
   optional: true
   run_if_changed: ^api/
```
Comment (hasueki) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.13.yaml:118]:
```suggestion
  skip_if_only_changed: (^(docs|examples|enhancements)/)|(^[A-Z]+\.md$)|((^|/)OWNERS$)
```

**Review by hasueki — CHANGES_REQUESTED** [review]
Few tweaks required

**Comment by hasueki** [comment]
/pj-rehearse pull-ci-openshift-hypershift-release-4.13-e2e-ibmcloud-iks

**Comment by hasueki** [comment]
/pj-rehearse pull-ci-openshift-hypershift-release-4.13-e2e-ibmcloud-roks

File: ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.13.yaml [hunk 6]
```diff
@@ -105,25 +105,23 @@ tests:
...
     env:
-      BOM_VERSION: 4.11.0_openshift
+      BOM_VERSION: 4.13.0_openshift
       MANAGED_CLUSTER_TYPE: openshift
-      MANAGEMENT_CLUSTER_KUBECONFIG: /etc/ibmcloud/secrets/roks.kubeconfig
-      TEST_TYPE: main
-      VERIFY_COMPLIANCE_OPERATOR_SCAN: "true"
+      MANAGEMENT_CLUSTER_KUBECONFIG: /etc/ibmcloud/secrets/roks-release.kubeconfig
+      TEST_TYPE: release
     workflow: hypershift-ibmcloud-e2e
```
Comment (hasueki) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.13.yaml:125]:
```suggestion
    workflow: hypershift-ibmcloud-e2e-release
```

**Comment by cappetersson** [comment]
/pj-rehearse pull-ci-openshift-hypershift-release-4.13-e2e-ibmcloud-iks
/pj-rehearse pull-ci-openshift-hypershift-release-4.13-e2e-ibmcloud-roks

**Comment by cappetersson** [comment]
/pj-rehearse pull-ci-openshift-hypershift-release-4.13-e2e-ibmcloud-iks

**Comment by cappetersson** [comment]
/pj-rehearse pull-ci-openshift-hypershift-release-4.13-e2e-ibmcloud-roks

**Comment by a-dsouza** [comment]
@csrwng @sjenning PTAL

**Pull request merged**